addons/vehicles/CfgVehicles/Viper.inc

```diff
@@ -0,0 +1,191 @@
+                class CommanderOptics: CommanderOptics {
+                    weapons[] = {
+                        "HMG_127_APC",
+                        "Laserdesignator_mounted",
+                        "SmokeLauncher"
+                    };
+                    magazines[] = {
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "200Rnd_127x99_mag_Tracer_Yellow",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "SmokeLauncherMag",
+                        "Laserbatteries"
+                    };
+                    class Components {
+                        class VehicleSystemsDisplayManagerComponentLeft: VehicleSystemsTemplateLeftCommander {
+                            class Components: components {
+                                class SensorDisplay {
+                                    componentType="SensorsDisplayComponent";
+                                    range[]={4000,2000,1000,8000};
+                                    resource="RscCustomInfoSensors";
+                                };
+                            };
+                        };
+                        class VehicleSystemsDisplayManagerComponentRight: VehicleSystemsTemplateRightCommander {
+                            defaultDisplay="SensorDisplay";
+                            class Components: components {
+                                class SensorDisplay {
+                                    componentType="SensorsDisplayComponent";
+                                    range[]={4000,2000,1000,8000};
+                                    resource="RscCustomInfoSensors";
+                                };
+                            };
+                        };
+                    };
+                };
+            };
+            weapons[] = {
+                "cannon_120mm_long",
+                "LMG_coax"
+            };
+            magazines[] = {
+                "32Rnd_120mm_APFSDS_shells_Tracer_Yellow",
+                "12Rnd_120mm_HEAT_MP_T_Yellow",
+                "12Rnd_120mm_HE_shells_Tracer_Yellow",
+                "4Rnd_120mm_LG_cannon_missiles",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow",
+                "200Rnd_762x51_Belt_Yellow"
+            };
+        };
+    };
+
+    textureList[] = {
+        "SWS_01", 1,
+        "SWS_02", 1,
+        "SWS_03", 1
+    };
+    class TextureSources {
+        class SWS_01 {
```

Comment:
```suggestion
        // TODO: Create an in-game change texture system to avoid this being the selected texture if Cobra isn't the team in it.
        class SWS_01 {
```
I'd like to loop around to this myself to add an event handler or ace interact to allow you to set the texture to your own variant so it's not just randomly selected if it isn't y'all in the vic.